Resources/Prototypes/_RMC14/Entities/Objects/Specific/chemical-containers.yml

```diff
@@ -0,0 +1,452 @@
+- type: entity
+  name: jug
+  parent: BaseItem
```

Comment:
you can use CMBeakerBase as parent here. Its located in `Resources/Prototypes/_RMC14/Entities/Objects/Medical/beaker.yml`
would be good if you moved your .yml file there too.